.travis.yml

```diff
@@ -3,13 +3,9 @@ language: ruby
 sudo: false
 
 rvm:
-  - 1.9.3
-  - 2.0.0
   - 2.1
-  - 2.2
   - jruby-19mode
-  - rbx-2
-  - ruby-head
+  - jruby-9.0.4.0
```

Comment:
I recently set our travis to `jruby-9000` and was surprised to see it running `9.0.0.0.pre1`.  I'm curious if you had any issues with JRuby 9k or specifying the version.